README.md

```diff
@@ -1,3 +1,157 @@
+5. **Code review**
+   1. Other developers should review code and leave suggestions. ([Tutorial](https://github.com/features/code-review/))
+      - No overly complex or clever code.
+      - Check for potential issues.
+      - Architectural improvements.
+      - Code is properly abstracted.
+      - Consider potential tests.
+   2. Code author is responsible for making any changes.
+      - Commit process follows the same process.
+   3. At least 2 reviewers should approve the code.
+      - Leave a comment with an emoji thumbs up.
+      - Briefly explain why it is okay.
+
+#### Minor bug fixes
+
+1. Update **development** and/or **feature** branches to the latest version.
+2. Checkout your target branch.
+3. Make Changes.
+4. Commit Changes.
+   - Prefix the message with: `[FIX]`.
+5. Push Changes.
+
+#### Miscellaneous
+
+**Exceptional work**
+
+Exceptional code will be show cased and discussed during a meeting to both reward the author and educate the other programmers.
+
+**Broken builds**
+
+When a build breaks there will be a public message in Discord noting who broke it. This is not to punish people but to ensure there is always some one who knows who to contact when the build is broken and or go to him/her to help fix it.
+
+No action will be taken against people who brake the build. However if it becomes a regular occurrence for an individual this will change. Action would vary case by case and is performed by the lead programmer.
+
+### Coding standards
+
+This project follows the a modified version of the Lefticus coding standard available on [GitHub](https://gist.github.com/lefticus/10191322) and the [C++ Code Guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md).
+
+Please read this page thoroughly before starting on your feature: [https://teamwisp.github.io/code_standards/](https://teamwisp.github.io/code_standards/)
+
+### Documentation standards
+
+To keep the codebase clear and understandable for all developers, we came up with some rules regarding the documentation developers have to make when changing the codebase.
+
+Please read this page thoroughly before making a pull request: [https://teamwisp.github.io/docs/](https://teamwisp.github.io/docs/)
+
+### Code architecture
+
+New contributors may find it a bit overwhelming to get started on a project of this size. This is why we have created a short introduction page on the Wiki that explains some of the basics of the framework: [https://teamwisp.github.io/architecture/](https://teamwisp.github.io/architecture/)
+
+## More information
+
+This README is only a summary of the most important things developers should know about the project. It is highly encouraged to have a look at our [Wiki](https://teamwisp.github.io/). It covers all topics mentioned above, and more...
+
+If you made it this far, congratulations! We hope to see you soon in our development server: https://discordapp.com/invite/KthSUvs
```

Comment:
Add a discord logo! it now fades away into all the text. Also maybe put the discord link a bit higher.

also I always hate it when people write stuff like this "If you made it this far, congratulations!"